<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_open_path">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>701</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_save_path">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>701</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start convert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mpeg_radio_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>MPEG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mjpeg_radio_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>MJPEG</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
